### PR TITLE
Improve classic theme integration

### PR DESCRIPTION
**Pull request by nihco2**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Classic : Layout fixes and css refactoring |
| Type? | improvement |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-840 |
| How to test? | Check the google doc link in JIRA |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: themes/classic/templates/_partials/header.tpl [hunk 1]
```diff
@@ -16,12 +16,12 @@
 {block name='header_top'}
     <div class="container">
       <div class="header-top row">
-        <div class="col-md-2">
+        <div class="col-md-1">
```
Comment (maximebiloe) [inline themes/classic/templates/_partials/header.tpl]: It makes the logo really really small 😕

File: themes/classic/_dev/js/components/product-select.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default class ProductSelect {
       $arrows.css('opacity', '.2');
     } else {
       $arrows.on('click', (event) => {
-        if ($(event.target).hasClass('arrow-up') && $('.js-modal-product-images').position().top < 0) {
+        if ($(event.target).hasClass('arrow-up') && $thumbnails.position().top < 0) {
```
Comment (maximebiloe) [inline themes/classic/_dev/js/components/product-select.js]: I think you forgot the parenthesis

File: themes/classic/templates/_partials/header.tpl (hunk 1 above)
Comment (nihco2) [inline themes/classic/templates/_partials/header.tpl]: Fixed here: https://github.com/PrestaShop/PrestaShop/pull/5774/files#diff-d1291278decbf0be8df2c9ef938c93a3R30

**Comment by maximebiloe** [comment]
boom

**Pull request merged**